src/util/createdb.cpp

```diff
@@ -98,9 +98,12 @@ int createdb(int argc, const char **argv, const Command& command) {
 
     size_t fileCount = filenames.size();
     DBReader<unsigned int>* reader = NULL;
+    DBReader<unsigned int>* hdrReader = nullptr;
     if (dbInput == true) {
         reader = new DBReader<unsigned int>(par.db1.c_str(), par.db1Index.c_str(), 1, DBReader<unsigned int>::USE_DATA | DBReader<unsigned int>::USE_INDEX | DBReader<unsigned int>::USE_LOOKUP);
         reader->open(DBReader<unsigned int>::LINEAR_ACCCESS);
+        hdrReader = new DBReader<unsigned int>((par.db1 + "_h").c_str(), (par.db1 + "_h.index").c_str(), 1, DBReader<unsigned int>::USE_DATA | DBReader<unsigned int>::USE_INDEX);
```

Comment:
par. hdr2 and hdr2Index might be better.

Reply:
Got it. I think I should be `par.hdr1` & `par.hdr1Index`